docs/faq.md

```diff
@@ -0,0 +1,72 @@
+# Frequently Asked Questions
+
+This page describes the most frequently asked questions about the Scorecard use, checks or relevance. These answers are continually changing. If you know any other question that could be interesting to be covered here, please [contribute](../CONTRIBUTING.md)!
+
+## Installation / Usage
+  - [Can I have a preview of the score?](#can-i-have-a-preview-of-the-score)
+  - [What is the difference between Scorecard and other Code Scanning Tools?](#what-is-the-difference-between-scorecard-and-other-code-scanning-tools)
+
+## Check-Specific Questions
+  - [Binary-Artifacts: Is it possible to set up a blocklist to check Binary-Artifacts?](#binary-artifacts-is-it-possible-to-set-up-a-blocklist-to-check-binary-artifacts)
+  - [Code-Review: Can I set Code-Review check to ignore bot commits?](#code-review-can-i-set-code-review-check-to-ignore-bot-commits)
+  - [Fuzzing: Scorecard accepts custom fuzzers and libfuzzer?](#fuzzing-scorecard-accepts-custom-fuzzers-and-libfuzzer)
+  - [Pinned-Dependencies: Will the scorecard see not-pinned dependencies in tests with Dockerfiles?](#pinned-dependencies-will-the-scorecard-see-not-pinned-dependencies-in-tests-with-dockerfiles)
+  - [Pinned-Dependencies: Can I use version pinning instead of hash pinning?](#pinned-dependencies-can-i-use-version-pinning-instead-of-hash-pinning)
+  - [Signed-Releases: Why would I sign releases?](#signed-releases-why-would-i-sign-releases)
+
+________________________________________________________________________________
+________________________________________________________________________________
+
+## Installation / Usage
+
+### Can I have a preview of the score?
+
+Yes, a preview of the Scorecard scores can be seen at https://api.securityscorecards.dev/projects/github.com/<username_or_org>/<repository_name>/ for the repositories tracked by the Scorecard Project for being considered relevant in the Open Source scenario.
+
+Anyhow, you can generate the scores for any public repository through the CLI following the steps bellow:
+
+1. [Installation](https://github.com/joycebrum/scorecard#installation)
+1. [Authentication](https://github.com/joycebrum/scorecard#authentication)
+1. [Basic Usage](https://github.com/joycebrum/scorecard#basic-usage)
+
+### What is the difference between Scorecard and other Code Scanning Tools?
+
+To determine exactly what are the differences it is important to analyse both tools and compare them. But, usually, the code scanning tools are focused on one or two specific types of vulnerabilities, while the Scorecard's Checks are focused on the overall security posture of the project. That's because the Scorecard is related to the Security Best Practices and whether the project is following them or not.
```

Comment:
Suggestion: omit the first sentence, since it doesn't really answer the question.